docs/how-to/convert-react-app.md

```diff
@@ -0,0 +1,24 @@
+---
+parent: How To
+nav_order: 1
+title: Convert React App to Modular Project
+---
+
+# Convert a React App to a Modular Project
+
+To convert an existing React app project into a Modular project, start by
+creating a new Modular project:
+
+` yarn create modular-react-app my-new-modular-project [--verbose] [--prefer-offline] [--repo]`
```

Comment:
I think you can remove the optional parameters as this is an example. You can link "creating a new Modular project" to the home page, so users can look at the whole command.